topics/sentinel.md

```diff
@@ -402,31 +405,66 @@ The following is a list of accepted commands:
 
 The following is an example of `SENTINEL SET` command in order to modify the `down-after-milliseconds` configuration of a master called `objects-cache`:
 
     SENTINEL SET objects-cache-master down-after-milliseconds 1000
 
-As already stated, `SENTINEL SET` can be used to set all the configuration parameters that are settable in the startup configuration file. Moreover it is possible to change just the master quorum configuration without removing and re-adding the master with `SENTINEL REMOVE` followed by `SENTINEL MONITOR`, but simply using:
+As already stated, `SENTINEL SET` can be used to set all the configuration parameters that are settable in the start-up configuration file. Moreover it is possible to change just the master quorum configuration without removing and re-adding the master with `SENTINEL REMOVE` followed by `SENTINEL MONITOR`, but simply using:
 
     SENTINEL SET objects-cache-master quorum 5
 
 Note that there is no equivalent GET command since `SENTINEL MASTER` provides all the configuration parameters in a simple to parse format (as a field/value pairs array).
 
+The following is an example of completely using the API to add a master to the
+sentinel, including the setting of the master's password.
+
+	SENTINEL MONITOR objects-cache-master 192.168.1.101 2
+    SENTINEL SET objects-cache-master auth-pass mysecretauthstring
+
+Handling Slave Removal
+---
+
+As Sentinel is designed to manage all replication configuration of every
+Redis instance it is informed of, either directly by adding a master or
+indirectly by an instance being a slave of a managed master, you can run
+into issues when you want a slave to no longer be part of a manages
+master pod. A key to understanding how this works is to know Sentinel
+really only knows what instances to manage by their `IP,PORT` address
+pair. Once Sentinel learns of an address pair it will forever try to
+manage it.
+
+Thus if you were to decide `C` now needs to be it's own master (ie. you
+are re-using instances) Sentinel will "correct" `C` after your `slaveof
+no one` command because it was told `C` is a slave to `A`.
+
+There are two ways to handle this. You can remove `A` from all sentinels
+managing it and re-add it, or you can "reset" the information for `A`.
+For remove-and-re-add you would of course use  `SENTINEL REMOVE A` on
+each sentinel and then repeat the `SENTINEL MONITOR` command to add it.
+However, during this time no events on `A` will be caught.
+
+The alternative is to issue `SENTINEL RESET A` on each master and let
+Sentinel re-discover the current slaves. This approach will leave `A` in
+a state where it will not fail over until it has a promotable slave, but
+the events should still be logged and caught - just not acted upon.
+
+
```

Comment:
We now have a similar section: http://redis.io/topics/sentinel#adding-or-removing-sentinels